sample/OSCON17/event

```diff
@@ -0,0 +1,78 @@
+{
+	"id": 3,
+	"name": "OSCON open source convention",
+	"latitude": 30.263569,
+	"longitude": -97.739606,
+	"location_name": "Austin Convention Center, Austin, TX ",
+	"start_time": "2017-05-08T13:00:00",
+	"end_time": "2017-105-11T20:00:00",
```

Comment:
@shreyasnbhat @mahikaw This is the reason you are not able to open schedule. It's a miracle Java Date class did not throw exception while parsing.

Please ensure all dates have at most 12 months